### PR TITLE
Towards 0.6.8.

### PR DESCRIPTION
**Pull request by sjrd**
FTR, the Towards 0.6.7 commit was e9b19e32fb8a96c8a14e57894e5054083f06194d

### CODE REVIEW

**Comment by sjrd** [comment]
Review by @nicolasstucki for fast-forward on master. Do not merge.

File: ir/src/main/scala/org/scalajs/core/ir/ScalaJSVersions.scala [hunk 1]
```diff
@@ -10,7 +10,7 @@ object ScalaJSVersions {
    */
 
   /** Scala.js version. */
-  val current: String = "0.6.7"
+  val current: String = "0.6.8-SNAPSHOT"
```
Comment (nicolasstucki) [inline ir/src/main/scala/org/scalajs/core/ir/ScalaJSVersions.scala]: I'm guessing that `binarySupported` was not updated on purpose. Is that right?

**Comment by nicolasstucki** [comment]
That is all. Otherwise LGTM.

File: ir/src/main/scala/org/scalajs/core/ir/ScalaJSVersions.scala (hunk 1 above)
Comment (sjrd) [inline ir/src/main/scala/org/scalajs/core/ir/ScalaJSVersions.scala]: Yes. 0.6.7 is forward binary compatible with 0.6.6. It emits the same version of the IR.

**Pull request merged**